### ISSUE TITLE
[BUG] Only "add user" works in admin options - JS error "SyntaxError: missing ) after argument list"

### ISSUE THREAD

**Issue by Epoxyde** [BoltAuth/Auth#38]
Hi,

There is a javascript issue in auth.twig causing an error: 

> SyntaxError: missing ) after argument list

I fixed it by replacing simple quotes with double quotes for setMessage arguments:
```
            boltExt.setMessage("useradd", "{{__('Adding user...')}}" );
            boltExt.setMessage("userdel", "{{__('Removing user(s)...')}}");
            boltExt.setMessage("userenable", "{{__('Enabling user(s)...')}}" );
            boltExt.setMessage("userdisable", "{{__('Disabling user(s)...')}}");
            boltExt.setMessage("roleadd", "{{__('Adding role...')}}");
            boltExt.setMessage("roledel", "{{__('Removing role...')}}");
            boltExt.setMessage("authnotsellHeader", "{{__('Nothing Selected!')}}");
            boltExt.setMessage("authnotsell", "{{__('You need to choose a auth.')}}");
            boltExt.setMessage("rolenotsellHeader", "{{__('None role Selected!')}}");
            boltExt.setMessage("rolenotsell", "{{__('You need to choose a role.')}}");
            boltExt.setMessage("autherrorHeader", "{{__('Error!')}}");
            boltExt.setMessage("autherror", "{{__('The server returned an error.')}}");
            boltExt.setMessage("confirmdeleteHeader", "{{__('Confim deletion')}}");
            boltExt.setMessage("confirmdelete", "{{__('Are you sure you want to delete these accounts?')}}");
            boltExt.setMessage("confirmdeleteButton", "{{__('Yes!')}}");
```

Details
-------

 - Bolt Auth Version: [ 3.0.5 ]
 - Bolt Version: [ 3.5.3 ]
 - [ Firefox Quantum for Ubuntu 60.0.2 ]

**Comment by artggd** [comment]
Are you browsing the admin in French? If so, then your issue should be fixed when #37 is merged 🙂

**Comment by Epoxyde** [comment]
@artggd Yes, you're right, I am browsing the admin in French. So rendez-vous to #37 merging. Thanks :-)
